gradle.properties

```diff
@@ -1,6 +1,6 @@
 # buildscript - project id
 projectGroup=com.generalbytes.batm.public
-projectVersion=1.0.18-SNAPSHOT
+projectVersion=1.0.19
```

Comment:
Why did you delete -SNAPSHOT?

Reply:
We discussed with Filip, that the versions are not snapshots - those are releases, in fact.